### ISSUE TITLE
format the string of form item label

### ISSUE THREAD

**Issue by shawn-show** [DevExpress/DevExtreme#14495]
the case is that :
I want to show the label of form item like 'CH<sub>4</sub>'
so I coded :
`
 label: {
                            text: 'CH' + '4'.sub()
                        },
`
but the result is 
`
<span class="dx-field-item-label-text">CH&lt;sub&gt;4&lt;/sub&gt;:</span>
`
 
how can I fixed the case?

**Comment by novsstation** [comment]
Hi,

Please pardon such a long delay in our response.
You can customize the editor and its label using the [template](https://js.devexpress.com/Documentation/Guide/Widgets/Form/Configure_Simple_Items/#Customize_a_Simple_Item) property.

```js
{
	dataField: "LastName",
	label: { visible: false },
	template: function (data, itemElement) {
		var label = $('<label></label>')
			.addClass('dx-field-item-label dx-field-item-label-location-left')
			.html('<span class="dx-field-item-label-text">CH<sub>4</sub>:</span>');
		itemElement.append(label);
		
		var editorContainer = $('<div></div>')
			.addClass('dx-field-item-content dx-field-item-content-location-right');

		var editor = $('<div></div>').dxTextBox({
			value: data.component.option('formData')[data.dataField],
			onValueChanged: function(e) {
				data.component.updateData(data.dataField, e.value);
			}
		});
		
		editorContainer.append(editor);
		itemElement
			.addClass('dx-flex-layout dx-label-h-align')
			.append(editorContainer);
	}            
}
```

I also created a [small sample](https://codepen.io/SNovikov/pen/JjKoyEL?editors=0010) for you.
Please check it and let me know if this solution helps.

**Comment by flagmanAndrew** [comment]
We implemented the label template in v22.2: [SimpleItem.label.template](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/#template).
